src/site/stages/build/plugins/modify-dom/index.js

```diff
@@ -8,16 +8,29 @@ const addSubheadingsIds = require('./add-id-to-subheadings');
 const checkBrokenLinks = require('./check-broken-links');
 const injectAxeCore = require('./inject-axe-core');
 
-const domModifiers = [
-  addNonceToScripts,
-  processEntryNames,
-  updateExternalLinks,
-  addSubheadingsIds,
-  checkBrokenLinks,
-  injectAxeCore,
-];
+const getDomModifiers = BUILD_OPTIONS => {
+  if (BUILD_OPTIONS.buildtype === 'vagovdev') {
```

Comment:
Any reason addNonceToScripts and checkBrokenLinks were remove here for dev builds? Is this related to the tests or hitching a ride?

Reply:
The `addNonceToScripts` and `checkBrokenLinks` `domModifiers` are being excluded because they're not needed for the unit tests.

Reply:
I was discussing the best way to flag those two modifiers as not-needed with Nick Sullivan and he suggested; “maybe check `BUILD_OPTIONS.BUILDTYPE` and skip if it’s `vagovdev`?”

Alternatively, I can add a property to the `options` that are passed in called something like ‘unit_testing_framework’ and set it to `true`?

Reply:
I updated the `BUILD_OPTIONS` to:
```
  const options = {
    buildtype: BUILDTYPE,
    entry: true,
    liquidUnitTestingFramework: true,
    // the following options are expected but the values don't matter
    host: 'host',
    port: 3001,
    protocol: 'http',
  };
```

and set the `domModifiers` array using `liquidUnitTestingFramework` instead of `buildtype`:
```
const getDomModifiers = BUILD_OPTIONS => {
  if (BUILD_OPTIONS.liquidUnitTestingFramework) {
    return [
      processEntryNames,
      updateExternalLinks,
      addSubheadingsIds,
      injectAxeCore,
    ];
  }

  return [
    addNonceToScripts,
    processEntryNames,
    updateExternalLinks,
    addSubheadingsIds,
    checkBrokenLinks,
    injectAxeCore,
  ];
};
```